kie-api/src/test/resources/META-INF/kie.conf.test1

```diff
@@ -0,0 +1 @@
+org.kie.api.internal.assembler.KieAssemblers = 3org.kie.api.internal.utils.MockAssemblersImpl
```

Comment:
I'm not sure I like the number before the component.

Why not using the marker annotation on the implementation as we discussed?